tools/editor/scene_tree_dock.cpp

```diff
@@ -1366,6 +1367,25 @@ void SceneTreeDock::_create() {
 
 		editor_data->get_undo_redo().create_action(TTR("Create Node"));
 
+		// Give the node a name with respect to the user's node name casing setting.
+		String new_name;
+		if (edited_scene) {
+			new_name = parent->validate_child_name(child);
+		} else {
+			new_name = child->get_type();
+		}
+		int name_casing = _get_node_name_casing("scenetree_editor/new_node_name_casing");
+		switch (name_casing) {
```

Comment:
It would be good to handle the case `NODE_NAME_CASING_PASCAL_CASE` for consistency, if only to just `break;` as there is nothing to do.